pkg/resources/config.go

```diff
@@ -79,3 +79,35 @@ var aliases = map[string]string{
 	"int2":    "smallint",
 	"uint":    "uint4",
 }
+
+var session_variables = []string{
```

Comment:
This is just used for validation within the resource. Not sure if we want to this just since it might be harder to keep things in sync.